### PR TITLE
Sync with nerves_system_rpi3

### PR DESCRIPTION
**Pull request by mobileoverlord**
Update project to sync with nerves-project/nerves_system_rpi3

### CODE REVIEW

**Comment by mobileoverlord** [comment]
I tested this locally 👍

File: example/lib/example/application.ex [hunk 1]
```diff
@@ -48,7 +48,7 @@ defmodule Example.Application do
     :os.cmd('udevadm trigger --type=devices --action=add')
     :os.cmd('udevadm settle --timeout=30')
 
-    System.put_env("QTWEBENGINE_CHROMIUM_FLAGS", "--disable-gpu")
+    System.put_env("QTWEBENGINE_CHROMIUM_FLAGS", "--disable-gpu --disable-dev-shm-usage")
```
Comment (fhunleth) [inline example/lib/example/application.ex:51]: This is an interesting change. Do you know what's up with it?

File: mix.exs [hunk 2]
```diff
@@ -83,8 +83,7 @@ defmodule KioskSystemRpi3.MixProject do
       "fwup-revert.conf",
       "fwup.conf",
       "LICENSE",
-      "linux-4.19.defconfig",
-      "logo_custom_clut224.ppm",
```
Comment (fhunleth) [inline mix.exs:87]: Did the splash screen logo go away? :(

**Review by fhunleth — APPROVED** [review]
Looks good on inspection. I didn't test.

**Pull request merged**